config-templates/salesforce/opportunities.yaml

```diff
@@ -0,0 +1,117 @@
+data_source:
+  name: salesforce_oppportunities
+  owners:
+    - support@transformdata.io
+  sql_table: salesforce.opportunity ### Update to your opportunities schema.table
+  mutability:
+    type: full_mutation
+    type_params:
+      update_cron: 0 10 * * *
+  identifiers:
+    - name: opportunity_id
+      type: primary
+      expr: id
+    - name: owner_id
+      type: foreign
+      expr: owner_id
+    - name: account_id
+      type: foreign
+  dimensions:
+    - name: ds
+      type: time
+      type_params:
+        is_primary: True
+        time_format: YYYY-MM-DD
+        time_granularity: day
+      expr: created_date::date
+    - name: close_date
+      type: time
+      type_params:
+        is_primary: False
+        time_format: YYYY-MM-DD
+        time_granularity: day
+    - name: lead_source
+      type: categorical
+    - name: campaign_name
+      type: categorical
+    - name: name
+      type: categorical
+    - name: stage_name
+      type: categorical
+    - name: is_closed
+      type: categorical
+  measures:
+    - name: opps
+      agg: count_distinct
+      expr: id
+    - name: closed_opps
+      agg: count_distinct
+      expr: case when is_closed = true then id else null end
+      create_metric: True
+    - name: won_opps
+      agg: count_distinct
+      expr: case when is_won = true then id else null end
+    - name: lost_opps
+      agg: count_distinct
+      expr: case when is_won = false and is_closed = true then id else null end
+    - name: amount
+      expr: amount_usd
+      agg: sum
+    
+
+---
+metric:
+  name: opps_created
+  type: measure_proxy
+  type_params:
+    measures:
+      - opps
+---
+metric:
```

Comment:
Possibly comment `closed_opps` out until we add the date functionality to tie to a metric - could be confusing to have `closed_opps` by `created_at` date.

Reply:
I actually think it's still a useful metric to have, and i assume people will want to see this by both closed and created. I like to look at this by created date because i can start to see when the line trend starts to stabilize - this also tells us what the average sales cycle looks like.